atlasdb-console/versions.lock

```diff
@@ -338,13 +347,13 @@
             ]
         },
         "io.dropwizard:dropwizard-jackson": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
             "transitive": [
                 "com.palantir.atlasdb:atlasdb-config"
             ]
         },
         "io.dropwizard:dropwizard-util": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
```

Comment:
yeah, I think we have to consolidate the Jackson & Dropwizard version bumps solely to the `timelock-server` module as downstream won't take a direct dependency on it. I don't know how hard it will be to make the current gradle module setup play nice with that, but I'm wondering if we will need to split `timelock-server` out into a separate repo, but defer to those with more gradle-fu than I.